### PR TITLE
DE-2698: es checker package update

### PR DESCRIPTION
**Pull request by setiat**
use `ts-ids-es-json-generator` package for ES-Checker

### CODE REVIEW

File: __tests__/integration/_test_generic_validator.py [hunk 1]
```diff
@@ -1,53 +0,0 @@
-from pathlib import Path
```
Comment (mlipschultz) [inline __tests__/integration/_test_generic_validator.py:1]: What's the reason for removing this test file?

Comment (setiat) [reply __tests__/integration/_test_generic_validator.py]: forgot to perform a `git add` after renaming.
Just fixed it.

Comment (setiat) [reply __tests__/integration/_test_generic_validator.py]: ignore the pR for now...
fixing build issues

**Comment by mlipschultz** [comment]
From [this commit](https://github.com/tetrascience/ts-ids-validator/pull/11/commits/57897f48d1523976ce77128fc9d2b2371a25d0f3) on a test branch and PR I made to investigate this, it looks like [`ids_es_json_generator` can't be found when trying to run it](https://app.travis-ci.com/github/tetrascience/ts-ids-validator/builds/240966806#L289).  I know that `ids_es_json_generator` is properly installed because it [runs successfully in that same build](https://app.travis-ci.com/github/tetrascience/ts-ids-validator/builds/240966806#L190).  I also confirmed that [`Path(...).read_text('UTF-8')`](https://github.com/tetrascience/ts-ids-validator/pull/11/commits/57897f48d1523976ce77128fc9d2b2371a25d0f3#diff-8269b02699463510434a5a0a0aacb1dda046b599bb17b27e87d7db9d126a61c4R62) will [read as text, not bytes](https://app.travis-ci.com/github/tetrascience/ts-ids-validator/builds/240966806#L291).  Therefore, I suspect that this is actually a difference in environments between running directly from the command-line vs. running through subprocess.

I did a quick check in [the docs](https://docs.python.org/3/library/subprocess.html) and:
> If _env_ is not `None`, it must be a mapping that defines the environment variables for the new process; these are used instead of the default behavior of inheriting the current process’ environment.

Unfortunately, `env` is None (the default), so that's a strike against my guess.  However, that does make me think that running `pipenv run python -m ids_es_json_generator ...` might be overkill and we just need `python -m ids_es_json_generator`.  [Making that change](https://github.com/tetrascience/ts-ids-validator/pull/11/commits/4a43e8a05fc10ee7cb1d718bb1147207d984c52e#diff-8269b02699463510434a5a0a0aacb1dda046b599bb17b27e87d7db9d126a61c4R98) does result in [all the tests passing](https://app.travis-ci.com/github/tetrascience/ts-ids-validator/builds/240967929#L1078).  Note that I do have another item in the .travis.yml file that's causing the build to fail, but that was just some debugging.

So requests:
* Please make all the changes I made in [`_create_new_elasticsearch_json`](https://github.com/tetrascience/ts-ids-validator/pull/11/files#diff-8269b02699463510434a5a0a0aacb1dda046b599bb17b27e87d7db9d126a61c4R96).  It fixes the problem and will help make debugging easier.
* In [`ElasticsearchChecker.run`](https://github.com/tetrascience/ts-ids-validator/pull/11/files#diff-8269b02699463510434a5a0a0aacb1dda046b599bb17b27e87d7db9d126a61c4R65), please use `repr(e)` instead of `str(e)` so it's clearer what exception was originally raised.  If we still need/want the stdout/stderr from `CalledProcessError`, then please have a specific except clause for it (like I do in that draft PR, although what you do with stdout, stderr can be different).

**Review by mlipschultz — APPROVED** [review]

**Pull request merged**